lib/imageUtils.js

```diff
@@ -315,6 +315,19 @@ function filterImages(imageElements) {
   });
 }
 
+/**
+ * Handle a single image.
+ * @param {string} imageSrc - The source of the image to handle.
+ *
+ */
+export function handleSingleImage(imageSrc) {
+  const imageElement = Array.from(document.querySelectorAll('img')).filter((img) => img.src === imageSrc)[0];
```

Comment:
if we have 2 images with the same src, it might not have the expected behaviour. Why don't you pass the image element instead of the element.src?

If you cannot pass an HTML element, let's try something else. Maybe the xPath or equivalent